pulser-core/pulser/backend/remote.py

```diff
@@ -98,23 +116,30 @@ def results(self) -> tuple[Result, ...]:
         """The actual results, obtained after execution is done."""
         return self._results
 
+    @property
+    def _submission_id(self) -> str:
+        """The same as the batch ID, kept for backwards compatibility."""
+        return self._batch_id
+
     @property
     def batch_id(self) -> str:
         """The ID of the batch containing these results."""
-        return self._submission_id
+        return self._batch_id
 
     @property
     def job_ids(self) -> list[str]:
-        """The IDs of the jobs within this results submission."""
+        """The IDs of the jobs within these results' submission."""
         if self._job_ids is None:
-            return self._connection._get_job_ids(self._submission_id)
+            return self._connection._get_job_ids(self._batch_id)
         return self._job_ids
 
     def get_status(self) -> SubmissionStatus:
```

Comment:
Yeah, I guess that will have to happen at some point. I just feel like this will be a mess to deprecate properly so I was avoiding doing that in this PR but if you tell me it should happen right away, I'll do it